content/en/docs/reference/command-line-tools-reference/feature-gates/apparmor.md

```diff
@@ -9,6 +9,10 @@ stages:
   - stage: beta
     defaultValue: true
     fromVersion: "1.4"
+    toVersion: "1.31"
```

Comment:
The `toVersion` need correction to reflect the last version for the previous stage.

```suggestion
    toVersion: "1.30"
```

Reply:
Done. Thanks!